test/e2e/federation-util-14.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Personally I would vote for putting contents of this file into:
test/e2e/framework/federation_util.go